crates/sources/src/sources/balancer_v2/graph_api.rs

```diff
@@ -9,21 +9,17 @@
 //!   from the node
 
 use super::swap::fixed_point::Bfp;
-use crate::{event_handling::MAX_REORG_BLOCK_COUNT, subgraph::SubgraphClient};
+use crate::{
+    event_handling::MAX_REORG_BLOCK_COUNT,
+    subgraph::{ContainsId, SubgraphClient},
+};
 use anyhow::{bail, Result};
 use ethcontract::{H160, H256};
 use reqwest::Client;
 use serde::Deserialize;
-use serde_json::json;
 use serde_with::{serde_as, DisplayFromStr};
 use std::collections::HashMap;
 
-/// The page size when querying pools.
-#[cfg(not(test))]
-const QUERY_PAGE_SIZE: usize = 1000;
-#[cfg(test)]
-const QUERY_PAGE_SIZE: usize = 10;
```

Comment:
Looks like the previous PR for the generic paginated query didn't have a smaller page size for tests.
Just wanted to point that out in case you actually wanted to keep it and overlooked it.

Reply:
I removed this because I didn't see a point for this at the moment. I think the original author was @nlordell so will wait for his comment if this is necessary or not :)

Reply:
It was to test pagination in the `#[ignore]` manual tests (because 1000 page size was hard to hit in real life).